crates/sui-core/src/unit_tests/transaction_manager_tests.rs

```diff
@@ -235,12 +250,35 @@ async fn transaction_manager_read_lock() {
         )
         .unwrap();
 
+    // Enqueue one transaction with two readable shared objects, one is above shared object.
```

Comment:
How about `Enqueue one transaction with two readonly shared object inputs, shared_object and shared_object_2?`